text/0000-bench-utils.md

```diff
@@ -0,0 +1,138 @@
+- Feature Name: black_box-and-clobber
+- Start Date: 2018-03-12
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC adds two functions to `core::mem`: `black_box` and `clobber`, which are
+mainly useful for writing benchmarks.
+
+# Motivation
+[motivation]: #motivation
+
+The implementation of these functions is backend-specific, and must be provided
+by the standard library.
```

Comment:
Probably worth adding *why* these are useful for benchmarks to the motivation section.

Reply:
Yeah, that was a bit scarce. I've expanded the motivation.